src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -82,7 +82,8 @@ void Terminal::Create(COORD viewportSize, SHORT scrollbackLines, IRenderTarget&
     _buffer = std::make_unique<TextBuffer>(bufferSize, attr, cursorSize, renderTarget);
     // Add regex pattern recognizers to the buffer
     // For now, we only add the URI regex pattern
-    _hyperlinkPatternId = _buffer->AddPatternRecognizer(R"([a-z]+[:.].*?(?=\s))");
+    std::wstring_view linkPattern{ LR"(\b(https?|ftp|file)://[-A-Za-z0-9+&@#/%?=~_|$!:,.;]*[A-Za-z0-9+&@#/%=~_|$])" };

```

Comment:
Should we be including `file` and `ftp` here, if we don't support opening them quite yet?

Reply:
i think we should

Reply:
I asked Dustin "why" and he said that we're unblocking these in a parallel PR. So I figure turning them off just to turn them back on shortly is annoying.